server/config/config.go

```diff
@@ -634,6 +634,8 @@ type ScheduleConfig struct {
 	MaxMergeRegionKeys uint64 `toml:"max-merge-region-keys" json:"max-merge-region-keys"`
 	// SplitMergeInterval is the minimum interval time to permit merge after split.
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval" json:"split-merge-interval"`
+	// SplitQPSThreshold is the qps threshold for split region.
+	SplitQPSThreshold uint64 `toml:"split-qps-threshold" json:"split-qps-threshold"`
```

Comment:
can we know this from TiKV? I think we can use `ConfigManager`.

Reply:
How to use `ConfigManager`？And I think we can also get `StoreHeartbeatInterval` and `RegionHeartbeatInterval` from it.

Reply:
What if this threshold is not the same between PD and TiKV?

Reply:
When the pd threshold is higher than the tikv threshold, some regions may not be merged as expected, and conversely, some regions that were just split by qps may be merged again, so we expect them to be the same, or not much different